src/controller/python/chip-device-ctrl.py

```diff
@@ -141,8 +137,7 @@ def FormatZCLArguments(args, command):
 
 class DeviceMgrCmd(Cmd):
     def __init__(self, rendezvousAddr=None, controllerNodeId=0, bluetoothAdapter=None):
-        self.lastNetworkId = None
-
+        self.lastNetworkId = None       
```

Comment:
Same as above, we need to be careful with line removals due to potential upstream issues